Composer/packages/client/src/pages/botProject/AppIdAndPassword.tsx

```diff
@@ -114,14 +103,23 @@ export const AppIdAndPassword: React.FC<AppIdAndPasswordProps> = (props) => {
   }, [projectId, mergedSettings, localMicrosoftAppId]);
 
   return (
-    <CollapsableWrapper title={formatMessage('App Id / Password')} titleStyle={title}>
+    <Fragment>
+      <div css={title}>{formatMessage('Microsoft App ID')}</div>
+      <div css={subtext}>
+        {formatMessage(
```

Comment:
Same comment as above. We shouldn't rely on `formatMessage` to add a space between the description and the Link.

Reply:
handled